locales/fr_FR/pcsx2_Main.po

```diff
@@ -1475,19 +1472,19 @@ msgstr "&Système"
 
 #: pcsx2/gui/MainFrame.cpp:365
 msgid "CD&VD"
-msgstr ""
+msgstr "CD&VD"
 
 #: pcsx2/gui/MainFrame.cpp:366
 msgid "&Config"
 msgstr "&Configuration"
 
 #: pcsx2/gui/MainFrame.cpp:367
 msgid "&Misc"
-msgstr "&Autres"
+msgstr "&Divers"
```

Comment:
Can't have D be the shortcut for both, it needs to be something different from the other things int he menu (&D means the shortcut is Alt+D)